src/server/api/routers/collection.ts

```diff
@@ -74,6 +74,21 @@ export const collectionRouter = createTRPCRouter({
       });
     }),
 
+  getSubmissionByCollectionId: publicProcedure //return collection if submitted, return null otherwise
+    .input(z.object({ collectionId: z.string(), userId: z.string() }))
+    .query(async ({ input: { collectionId, userId } }) => {
```

Comment:
I think ctx is a field like input here or it might be the second argument of the callback but it will give u the userid